client/static/robots.txt

```diff
@@ -0,0 +1,2 @@
+User-agent: *
+Disallow: /
```

Comment:
Makes sense for now; we'll need to readdress closer to launch -- might be worth a ticket?

Reply:
https://github.com/usds/justice40-tool/issues/84